yarn.lock

```diff
@@ -1384,22 +1384,6 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
-"@fluentui/date-time-utilities@^7.9.0":
```

Comment:
this is wrong. you need to update the package names in root `package.json` file as well